DOCUMENTATION.md

```diff
@@ -696,9 +696,9 @@ elements you'll touch in a normal application are:
   internal implementation of Optimistic Concurrency Control / retry loop used
   by `Stream`. It's recommended to at least scan this file as it defines the
   Transaction semantics everything is coming together in service of.
-- [`type Stream`](https://github.com/jet/equinox/blob/master/src/Equinox/Equinox.fs#L11) -
+- [`type Decider`](https://github.com/jet/equinox/blob/master/src/Equinox/Decider.fs#L11) -
   surface API one uses to `Transact` or `Query` against a specific stream
-- [`type Target` Discriminated Union](https://github.com/jet/equinox/blob/master/src/Equinox/Equinox.fs#L42) -
+- [`type Target` Discriminated Union](https://github.com/jet/equinox/blob/master/src/Equinox/Decider.fs#L42) -
   used to identify the Stream pertaining to the relevant Aggregate that
   `resolve` will use to hydrate a `Stream`
```

Comment:
Same

Reply:
that `Target` was actually way out of date, thanks for flagging 😊